LINUX.md

```diff
@@ -480,6 +480,39 @@ plugins=(gitfast last-working-dir common-aliases zsh-syntax-highlighting history
 
 ## Installing Python (with [`pyenv`](https://github.com/pyenv/pyenv))
 
+### Uninstall `conda`
+
+As we are using `pyenv` to install and manage our Python version, we need to uninstall [`conda`](https://docs.conda.io/projects/conda/en/latest/), another package manager you may have on your machine if you previously installed [Anaconda](https://www.anaconda.com/). Thus, we are preventing any possible Python version issue later.
+
+Check if you have `conda` installed on your machine:
+```bash
+conda list
+```
+If you have `zsh: command not found: conda`, you can **skip** the uninstall of `conda` and jump to the **Install `pyenv`** section.
+
+<details>
+    <summary markdown='span'><code>conda</code> uninstall instructions</summary>
+
+- Install the Anaconda-Clean package from your terminal and run the cleaning
+```bash
+conda install anaconda-clean
+anaconda-clean --yes
+```
+- Remove every Anaconda directories
+```bash
+rm -rf ~/anaconda2
+rm -rf ~/anaconda3
+```
+- Remove Anaconda path from your `.bash_profile`
+    - Open the file with `code ~/.bash_profile`
+    - If the file opens find the line matching the following pattern `export PATH="/path/to/anaconda3/bin:$PATH"` and delete the line
+    - Save the file with `CTRL` + `s`
+- Restart your terminal with `exec zsh`
```

Comment:
Since my shell is `zsh`, I didn't have anything in my `.bash_profile` but got these lines added to my `.zshrc` instead: 

```bash
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/home/dmilon/anaconda3/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/home/dmilon/anaconda3/etc/profile.d/conda.sh" ]; then
        . "/home/dmilon/anaconda3/etc/profile.d/conda.sh"
    else
        export PATH="/home/dmilon/anaconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
```

Maybe we could add an extra bullet point list with instructions for `zsh` :thinking: